manifest.go

```diff
@@ -567,6 +584,87 @@ func ReadManifestList(in io.Reader) ([]ManifestFile, error) {
 	return out, dec.Error()
 }
 
+// WriteManifestList writes a list of v2 manifest files to an avro file.
+func WriteManifestList(out io.Writer, files []ManifestFile) error {
+	var version int
+	if len(files) > 0 {
+		version = files[0].Version()
+	} else {
+		// No files to write
+		return nil
+	}
```

Comment:
let's simplify this a little

```suggestion
        if len(files) == 0 {
                return nil
        }
        
        version := files[0].Version()
```